cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml

```diff
@@ -82,17 +92,14 @@ spec:
               fi;
       # BEGIN_PROMETHEUS_TO_SD
       - name: prometheus-to-sd-exporter
-        image: gcr.io/google-containers/prometheus-to-sd:v0.1.3
+        image: gcr.io/google-containers/prometheus-to-sd:v0.2.1
```

Comment:
Unfortunately this version have some problems related to noisy logging. I have just released v0.2.2, please switch to it.